plugins/ScheduledReports/API.php

```diff
@@ -473,6 +473,10 @@ public function generateReport($idReport, $date, $language = false, $outputType
             }
         }
 
+        usort($processedReports, function ($a, $b) {
+            return $a['metadata']['uniqueId'] <=> $b['metadata']['uniqueId'];
+        });
+
```

Comment:
This was added to get the scheduled reports tests to pass as the sort order of the reports on Travis was different for PHP 7 vs PHP 8. It seems this might be a general issue not related to this PR, I'll revert it and retry the build.

Reply:
Removing this sort causes the scheduled reports tests to fail again, added it back and the tests pass.